src/Pos/Generator/Block/Mode.hs

```diff
@@ -117,6 +125,7 @@ data BlockGenContext = BlockGenContext
     , bgcSystemStart       :: !Timestamp
     , bgcParams            :: !BlockGenParams
     , bgcDelegation        :: !DelegationVar
+    , bgcCustomUtxo        :: !Utxo
```

Comment:
Yes, but it should be called `npGenesisUtxo` there. It had to be renamed in November, but nobody had time since then. Let's stop copying nonsensical stuff.